tests/test_wmb.py

```diff
@@ -0,0 +1,151 @@
+import sys
+import pytest
+import pytz
+import csv
+import random
+from sqlalchemy import and_
+
+from datetime import datetime
+from decimal import Decimal
+from io import StringIO
+from tempfile import TemporaryFile
+
+from pycds import Obs, History, Network, Station
+from crmprtd.wmb import ObsProcessor, insert_obs, check_history
+
+
+@pytest.mark.parametrize(('val', 'hid', 'd', 'vars_id', 'expected'), [
+    # It looks like the vars table is indexed?
+    (2.7, 20, datetime(2016, 9, 13, 6, tzinfo=pytz.utc), Decimal(1), 4)
+])
+def test_insert_obs(test_session, val, hid, d, vars_id, expected):
+    insert_obs(val, hid, d, vars_id, test_session)
+    q = test_session.query(Obs)
+    assert q.count() == expected
```

Comment:
So I'm not sure that there's any value to parametrizing the "expected" value. Isn't the result 4, just the number of observations that are already in the test_session when you get it (3) plus the number that you have inserted (1)? In that case, that number will never change with any other parameters to this test. What may be more valuable is to count the observations in the database before and after running `insert_obs()` and assert that the count after == the count before + 1.